### PR TITLE
Improve app article loading

### PR DESCRIPTION
**Pull request by pikkuruusu**
What this does:
• Uses React.lazy for the most read list component (don't think this do much when we return the page from the server)
• Lazy loads all images except main header image. Uses react-lazy-load-component for this.
• Use smaller file formats for fonts, woff2 seems to have been supported for quite a while now.
• We seemed to be loading some css twice. Stopped doing that.

### CODE REVIEW

File: apps/app-article-server/src/browser/components/article.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
-import React, { Component } from "react";
+import React, { Component, Suspense } from "react";
 import hblDefaultImage from "../assets/images/hbl-fallback-img.png";
 import Header from "./header";
 import Additional from "./article-additional";
 import ArticleDetails from "./article-details";
 import Content from "./article-content";
-import MostReadArticles from "./most-read-articles";
+// import MostReadArticles from "./most-read-articles";
```
Comment (jllang) [inline apps/app-article-server/src/browser/components/article.js:7]: Do we need this comment?

File: apps/app-article-server/src/browser/index.css [hunk 2]
```diff
@@ -1,21 +1,27 @@
 @font-face {
   font-family: "Roboto";
-  src: url("https://cdn.ksfmedia.fi/assets/fonts/Roboto-Regular.ttf");
+  src: url("https://cdn.ksfmedia.fi/assets/fonts/Roboto-Regular-webfont.woff2") format("woff2");
```
Comment (jaakkoha-bytecraft) [inline apps/app-article-server/src/browser/index.css:3]: We could also include both of the fonts here and let the browser decide.

The format is something around:
``` 
src:  url('Montserrat-Regular.woff') format('woff'), 
         url('Montserrat-Regular.ttf')  format('truetype'),
         url('Montserrat-Regular.svg#Montserrat') format('svg');
```

File: apps/app-article-server/src/browser/components/mobile-article-list.js [hunk 3]
```diff
@@ -64,14 +65,18 @@ const MobileList = (props) => {
...
+                    <>
+                    <LazyLoadImage
                       src={item.listImage.url.includes("imengine")
-                          ? item.listImage.url + "&function=hardcrop&width=360&height=200&q=65"
-                          : item.listImage.tinyThumb
-                          }
-                      alt=""
+                      ? item.listImage.url + "&function=hardcrop&width=360&height=200&q=65"
+                      : item.listImage.tinyThumb
+                      }
```
Comment (jllang) [inline apps/app-article-server/src/browser/components/mobile-article-list.js:73]: I don't know what kind of JS style convention we're following, but my intuition tells to use indentation here.

**Review by jllang — APPROVED** [review]

File: apps/app-article-server/src/browser/components/mobile-article-list.js (hunk 3 above)
Comment (jaakkoha-bytecraft) [reply apps/app-article-server/src/browser/components/mobile-article-list.js:73]: `npx prettier --write src/browser/components/mobile-article-list.js` formats stuff nicely.

File: apps/app-article-server/src/browser/index.css (hunk 2 above)
Comment (pikkuruusu) [reply apps/app-article-server/src/browser/index.css:3]: True, I'll add that.

File: apps/app-article-server/src/browser/components/mobile-article-list.js (hunk 3 above)
Comment (pikkuruusu) [reply apps/app-article-server/src/browser/components/mobile-article-list.js:73]: Good catch, I'll fix it.

**Pull request merged**